docs-src/zdm-core/modules/migrate/partials/interactive-lab.adoc

```diff
@@ -1,3 +1,5 @@
+Now that you've seen a conceptual overview of the process, why not putting what you learned in practice? 
+
 We've built a complementary learning resource that is a companion to this comprehensive ZDM documentation. It's the {zdm-product} Interactive Lab, available for you here:
 
 link:https://www.datastax.com/dev/zdm[https://www.datastax.com/dev/zdm, window="_blank"]
```

Comment:
```suggestion
https://www.datastax.com/dev/zdm[,window="_blank"]
```

Reply:
I like the conversational "Now that you've..." line but will make a minor edit.  I'll also change the L5 link as you proposed (thanks).  That will come in the next commit. Resolving for now without this commit.

Reply:
Updated in latest commit.